backstop/tests/modal.html

```diff
@@ -40,13 +40,21 @@ <h1>Modal</h1>
 
     <a
       id="open-modal"
-      onclick="openModal()"
+      onclick="toggleModal('myModal')"
       class="iui-anchor"
     >Open modal</a>
 
+    <br>
+
+    <a
+      id="open-full-page-modal"
+      onclick="toggleModal('fullPageModal')"
+      class="iui-anchor"
+    >Open full page modal</a>
+
     <script>
```

Comment:
Can you move this whole script to the bottom of body as we have in all others HTMLs?

Reply:
Done